ansible/inventory-sample.yml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
We should stick to what we had on the previous sample, it has the same format as the inventory generated automatically by the tool.